helm/generic-node/templates/generic-node.yaml

```diff
@@ -0,0 +1,199 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Values.name}}
+  labels:
+    app: {{.Values.name}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.Values.name}}
+  template:
+    metadata:
+      labels:
+        app: {{.Values.name}}
+        testnet: {{.Values.testnetName}}
+        role: seed
+        class: "generic-node"
+        version: {{.Values.image.tag | quote}}
+        syncStatus: INIT
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: {{.Values.ports.metrics | quote}}
+        prometheus.io/path: '/metrics'
+    spec:
+      {{- if .Values.libp2p.staticKeypair.enabled}}
+      initContainers:
+      - name: libp2p-init-my-service
+        image: busybox:1.28
+        command: ['sh', '-c', '/bin/cp /libp2p-keys/* /root/libp2p-keys && /bin/chmod -R 0700 /root/libp2p-keys/']
+        volumeMounts:
+        - name: source-libp2p-keys
+          mountPath: /libp2p-keys 
+        - name: destination-libp2p
+          mountPath: /root/libp2p-keys
+      {{- end }}
+      containers:
+      - name: daemon
+        image: "{{.Values.image.repository }}:{{.Values.image.tag}}"
+        imagePullPolicy: {{.Values.image.pullPolicy}}
+        {{- if .Values.features.customEntrypoint.enabled}}
+        command: [ {{ .Values.features.customEntrypoint.command}} ]
+        {{- end }}
+
+        resources:
+          limits:
+          requests:
+            memory: "32Gi"
+            cpu: "12"
+        args:
+        - "daemon"
+        - "--log-level"
+        - "Debug"
+        - "--log-json"
+        - "--log-snark-work-gossip"
+        - "true"
+        - "--config-directory"
```

Comment:
Remove --config-directory and - /root/.features-config, the default config directory will be fine for this chart